contracts/test/StakeTable.t.sol

```diff
@@ -530,4 +553,162 @@ contract StakeTable_Test is Test {
 
         assertEq(abi.encode(node), abi.encode(nullNode));
     }
+
+    // Queue logic
+
+    function getQueueParameters(QueueType queueType) public view returns (uint64, uint64) {
+        if (queueType == QueueType.Registration) {
+            return (
+                stakeTable.firstAvailableRegistrationEpoch(),
+                stakeTable.pendingRegistrationsInFirstAvailableRegistrationEpoch()
+            );
+        } else {
+            return (
+                stakeTable.firstAvailableExitEpoch(),
+                stakeTable.pendingExitsInFirstAvailableExitEpoch()
+            );
+        }
+    }
+
+    /// @dev Helper function to check the queue parameters depending on the queue type
+    function checkQueueParameters(
+        QueueType queueType,
+        uint64 expectedFirstAvailableEpoch,
+        uint64 expectedPendingRequests
+    ) private {
+        (uint64 v, uint64 w) = getQueueParameters(queueType);
+
+        assertEq(v, expectedFirstAvailableEpoch);
+        assertEq(w, expectedPendingRequests);
+    }
+
+    /// @dev Helper function to be able to fuzz with the type of queue
+    function asQueueType(uint256 n) private view returns (QueueType) {
+        QueueType queueType;
+        uint256 typeOfQueueInt = bound(n, 0, 1);
+        if (typeOfQueueInt == 0) {
+            queueType = QueueType.Registration;
+        } else if (typeOfQueueInt == 1) {
+            queueType = QueueType.Exit;
+        } else {
+            revert("Queue type not supported");
+        }
```

Comment:
we will never hit this, as `typeOfQueueInt` is already bound to be `[0,1]`

Reply:
The corresponding function has been deleted in 8038961f5e4df59b51bcaab10f3581eba25f8e89.